tox.ini

```diff
@@ -1,22 +1,17 @@
-# Operator charm: tox.ini
+# Source charm: ./tox.ini
+# This file is managed centrally by release-tools and should not be modified
+# within individual charm repos.  See the 'global' dir contents for available
+# choices of tox.ini for OpenStack Charms:
+#     https://github.com/openstack-charmers/release-tools
```

Comment:
I think we should probably consider a specific tox.ini that is shared for the k8s operator charms rather than adopting all the same cruft from what we currently have.

Reply:
Done